examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -64,36 +64,66 @@
 
 
 def save_model_card(
-    repo_id: str, images=None, base_model=str, train_text_encoder=False, prompt=str, repo_folder=None, vae_path=None
+    repo_id: str,
+    images=None,
+    base_model=str,
+    train_text_encoder=False,
+    instance_prompt=str,
+    validation_prompt=str,
+    repo_folder=None,
+    vae_path=None,
 ):
-    img_str = ""
+    img_str = "widget:\n" if images else ""
     for i, image in enumerate(images):
         image.save(os.path.join(repo_folder, f"image_{i}.png"))
-        img_str += f"![img_{i}](./image_{i}.png)\n"
+        img_str += f"""
+        - text: '{validation_prompt if validation_prompt else ' ' }'
+          parameters:
+            negative_prompt: '-'
```

Comment:
```suggestion
```
we can leave this blank if there's no negative prompt

Reply:
for some reason, it was breaking when I removed it

Reply:
Check out how a non-breaking version works without it: https://huggingface.co/e-n-v-y/envy-awesomizer-xl-01/blob/main/README.md?code=true#L24

Reply:
committing a fix now